src/graph/backend/dnnl/kernels/sdp_primitive.cpp

```diff
@@ -142,13 +142,6 @@ void sdp_primitive_kernel_t<quantized>::prepare_args_set(
         mem_idx.first.set_data_handle(
                 outputs[mem_idx.second].get_data_handle());
     }
-
-    grantor_t var_grantor = memory_planner_.internal_temporary_grantor(
```

Comment:
could you explain why remove these code?
Do we need to check if scratchpad is needed ?